BUILDING.md

```diff
@@ -10,30 +10,34 @@ When using the name 'version' we mean the versioning scheme described in [VERSIO
 
 This document is to describe the functionality a project MUST provide in terms of creating build artifacts. It also describes the structure in which project's MUST write build artifacts in.
 
-We propose:
+a project MUST provide:
+
  - a folder name convention for build artifacts
  - a folder structure for the above-mentioned build artifacts folder
- - a list of platforms we will target
- - Using docker-compose with a service for each build target
+ - a list of targets
+ - a file called `bin/build.{target}.{ext}` to target each of the build targets
```

Comment:
is `build` a generic or literal name here?

Reply:
literal: `bin/build.windows.bat`